spec/fixtures/Puppetfile

```diff
@@ -1,4 +1,5 @@
 mod 'homebrew',    '0.0.16', :github_tarball => 'boxen/puppet-homebrew'
+mod 'brewcask',    '0.0.6',  :github_tarball => 'toolbear/puppet-brewcask'
```

Comment:
This change depends on boxen/puppet-brewcask#33. Once merged, that'll go back to original upstream. This PR is essentially blocked on the brewcask PR.